from core.exceptions import CustomException 
 
 
# class LogsDuplicateException(CustomException): 
#     code = 400 
#     error_code = 10000 
#     message = "duplicate logs name" 
 
 
# class LogsNotFoundException(CustomException): 
#     code = 404 
#     error_code = 10000 
#     message = "logs not found" 
